acousticbrainz/views/data.py

```diff
@@ -44,21 +46,21 @@ def view_high_level(mbid):
 
 @data_bp.route("/<uuid:mbid>", methods=["GET"])
 def summary(mbid):
-    lowlevel, highlevel, genres, moods, other = get_summary_data(mbid)
-
-    info = _get_track_info(mbid, lowlevel['metadata'])
-
+    summary = get_summary_data(mbid)
+    
+    info = _get_track_info(mbid, summary['lowlevel']['metadata'] if summary['lowlevel'] else None)
     # Tomahawk player stuff
-    if not ('artist' in lowlevel['metadata']['tags'] and 'title' in lowlevel['metadata']['tags']):
+    if not ('artist' in info and 'title' in info):
```

Comment:
It's not directly related to your change, but the tomahawk_url code should be a separate method, taking `metadata` as an argument